README.md

```diff
@@ -98,23 +98,10 @@ We want to make sure VOLK compiles on a wide variety of compilers. Thus, we targ
 
 ## License
 
->
-> Copyright 2015 Free Software Foundation, Inc.
->
-> This file is part of VOLK
->
-> VOLK is free software; you can redistribute it and/or modify
-> it under the terms of the GNU General Public License as published by
-> the Free Software Foundation; either version 3, or (at your option)
-> any later version.
->
-> VOLK is distributed in the hope that it will be useful,
-> but WITHOUT ANY WARRANTY; without even the implied warranty of
-> MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-> GNU General Public License for more details.
->
-> You should have received a copy of the GNU General Public License
-> along with GNU Radio; see the file COPYING.  If not, write to
-> the Free Software Foundation, Inc., 51 Franklin Street,
-> Boston, MA 02110-1301, USA.
->
+VOLK is moving from the GNU General Public License v3.0 or later (GPL-3.0-or-later) 
+to the GNU Lesser General Public License v3.0 or later (LGPL-3.0-or-later).
+At this point in time, much of the code in the repository is still GPL-licensed, 
+but new contributors are asked to use the LGPLv3+ for their code contributions. 
+Existing contributors are very kindly requested to also allow LPGL-licensing by 
```

Comment:
`to also allow LPGL-licensing by` -> `to relicense their GPL-3.0-or-later license code contributions to LGPL-3.0-or-later by`

Reply:
```suggestion
Existing contributors are very kindly requested to re-submit their GPL-3.0-or-later license code contributions to LGPL-3.0-or-later by
```
After various discussion, we ask everyone to re-submit their code. However, all contributors can re-submit by adding themselves to the list instead of actually opening a new PR with all their commits. That would be infeasible to handle.